taxonomy-colleges.php

```diff
@@ -125,7 +125,7 @@
 		<div class="container my-5">
 			<div class="row">
 				<div class="col-lg-8">
-					<h2><?php echo ( $news_title = $term->colleges_alias ) ? $news_title . " News" : str_replace( 'College of ', '', $term->name ) . " News"; ?></h2>
+					<h2><?php echo ( $news_title = get_field( 'colleges_alias', 'colleges_' . $term->term_id ) ) ? $news_title . " News" : str_replace( 'College of ', '', $term->name ) . " News"; ?></h2>
```

Comment:
This logic isn't new, but the `str_replace` here won't work well for Rosen College of Hospitality Management--maybe we should just get rid of it (e.g. just print "News" if the alias isn't set).  Alternatively, maybe we fall back to the full college name.